### PR TITLE
[4.0] Inject the dependencies in the stats module

### PR DESCRIPTION
**Pull request by laoneo**
Inject the dependencies in the stats module. Additionally some logic in the layout got moved to the module class to simplify the layout.

### CODE REVIEW

**Comment by PhilETaylor** [comment]
I have not tested this item.<br /><br />Well without testing instructions its hard to know if it passed test

I applied the PR and nothing broken - so I guess thats good. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22902">issues.joomla.org/tracker/joomla-cms/22902</a>.</sub>

**Comment by laoneo** [comment]
Just open the back end cpanel and then it should work as before.

File: administrator/modules/mod_status/mod_status.php [hunk 1]
```diff
@@ -31,6 +34,20 @@
...
+
+$multilanguageStatusModuleOutput = '';
+
+// Check if the multilangstatus module is present and enabled in the site
+if (class_exists(MultilangstatusAdminHelper::class) && MultilangstatusAdminHelper::isEnabled($app, $db))
+{
+	// Publish and display the module
+	MultilangstatusAdminHelper::publish($app, $db);
+
+	if (Multilanguage::isEnabled($app, $db)) {
```
Comment (Quy) [inline administrator/modules/mod_status/mod_status.php]:
```suggestion
	if (Multilanguage::isEnabled($app, $db)) 
	{
```

**Comment by bonzani** [comment]
I have tested this item :white_check_mark: successfully on 9be335640d5e5b24a49dd5d3d9e6c8a02d020a97<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22902">issues.joomla.org/tracker/joomla-cms/22902</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**